frontend/src/components/Buttons/SortButton.vue

```diff
@@ -1,63 +1,82 @@
 <template>
+  <v-btn icon="mdi-heart" @click="changeOrder">
+    <v-icon v-if="ascendingOrder">
+      <i-mdi-sort-ascending />
+    </v-icon>
+    <v-icon v-else>
+      <i-mdi-sort-descending />
+    </v-icon>
+  </v-btn>
   <v-menu offset-y>
-    <template #activator="{ on, attrs }">
+    <template #activator="{ props }">
       <v-btn
         v-if="!$vuetify.display.smAndDown"
         class="my-2"
         variant="text"
         rounded
         :disabled="disabled"
-        v-bind="attrs"
-        v-on="on">
-        {{ $t('sortByType', { type: items[model].name }) }}
+        v-bind="props">
+        {{
+          $t('sortByType', {
+            type:
+              model == 0
+                ? items[0].title
+                : items.find((x) => x.value == model[0]).title
+          })
+        }}
         <v-icon end>
           <i-mdi-menu-down />
         </v-icon>
       </v-btn>
-      <v-btn
-        v-else
-        :disabled="disabled"
-        class="my-2"
-        icon
-        v-bind="attrs"
-        v-on="on">
-        <v-icon>
-          <i-mdi-sort-alphabetical-ascending />
+      <v-btn v-else :disabled="disabled" class="my-2" icon v-bind="props">
+        <v-icon v-if="model == 0 || model[0] == items[0].value">
+          <i-mdi-sort-alphabetical-variant />
+        </v-icon>
+        <v-icon v-else-if="model[0] == items[1].value">
+          <i-mdi-numeric-9-plus-box-outline />
+        </v-icon>
+        <v-icon v-else-if="model[0] == items[2].value">
+          <i-mdi-calendar-range />
         </v-icon>
       </v-btn>
     </template>
-    <v-list dense>
-      <v-list-group
-        v-model="model"
-        @change="$emit('change', items[model].value)">
-        <v-list-item v-for="item in items" :key="item.value">
-          <v-list-item-title>{{ item.name }}</v-list-item-title>
-        </v-list-item>
-      </v-list-group>
-    </v-list>
+    <v-list
+      v-model:selected="model"
+      :items="items"
+      class="filter-content"
+      @update:selected="emit('change', model[0], descendingOrder)" />
```

Comment:
As below, this seems wrong

Reply:
see https://github.com/jellyfin/jellyfin-vue/pull/1865#discussion_r1087739438